pandas/core/internals/construction.py

```diff
@@ -415,6 +420,10 @@ def _homogenize(data, index, dtype: Optional[DtypeObj]):
                 # Forces alignment. No need to copy data since we
                 # are putting it into an ndarray later
                 val = val.reindex(index, copy=False)
+            # extract_array should be preferred? But that gives failures for
+            # `extension/test_numpy.py`
+            # val = extract_array(val, extract_numpy=True)
+            val = val._values
```

Comment:
cc @jbrockmendel this is a PITA .. (similar as in https://github.com/pandas-dev/pandas/pull/39722#discussion_r573896611)

Reply:
i very much agree.  id just get rid of the tests that patch PandasArray._typ altogether.

in create_block_manager_from_arrays and make_block we specifically check for ABCPandasArray, could mirror the usage there for now

Reply:
>  id just get rid of the tests that patch PandasArray._typ altogether.

Which means getting rid of `extension/test_numpy.py` altogether? 
(the tests that only test the EA not inside a DataFrame or Series or so will pass, but I don't think it is worth maintaining a skips/xfails for this)

Reply:
> Which means getting rid of extension/test_numpy.py altogether?

i expect some of it could be salvaged, but a lot of it would become pointless, yah

Reply:
OK to leave this for a separate issue/PR? (will open an issue to discuss this problem)

Reply:
> `TODO extract_array should be preferred`

im actually OK with `._values` when we know we have a Series, as it will be somewhat more performant.

> will open an issue to discuss this problem

pls add the GH ref to the comment

> OK to leave this for a separate issue/PR? 

fine by me

Reply:
Opened https://github.com/pandas-dev/pandas/issues/40021, will add a comment about it

Reply:
i think  can "resolve conversation"?